pytestqt/qtbot.py

```diff
@@ -334,12 +345,18 @@ def waitSignals(self, signals=None, timeout=1000, raising=None):
 
         .. note:: This method is also available as ``wait_signals`` (pep-8 alias)
         """
+        assert force_order in ["none", "simple", "strict"], "force_order has to be set to 'none', 'simple' or 'strict'"
```

Comment:
This should be a `if` raising a `ValueError` - assertions are deactivated with `-O` for example, and an `AssertionError` is a weird thing to get for invalid parameters.